includes/Core/CLI/Reset_CLI_Command.php

```diff
@@ -27,16 +28,29 @@ class Reset_CLI_Command extends CLI_Command {
 	 *
 	 * ## OPTIONS
 	 *
+	 * [--persistent]
+	 * : Remove persistent site kit options too.
+	 *
 	 * ## EXAMPLES
 	 *
 	 *     wp google-site-kit reset
+	 *     wp google-site-kit reset --persistent
 	 *
 	 * @since 1.11.0
+	 * @since n.e.x.t Added --persistent flag to delete persistent options.
+	 *
+	 * @param Array $args Args.
+	 * @param Array $assoc_args Additional flags.
```

Comment:
The type should be lowercase here, otherwise an IDE will think it's a class.
```suggestion
	 * @param array $args       Positional args.
	 * @param array $assoc_args Additional flags.
```